tag_dedupe/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM gcr.io/cloud-builders/gcloud
+
+COPY dedupe.py /
+
+# This is the file containing the auth information for your service account. 
+# Necessary inside the container until gcloud supports cross container authentication when using cloudbuild.
+# Once this is addressed, remove this and update dedupe.py accordingly.
+COPY auth.json /
```

Comment:
ah good idea. this would get rid of the possibility of using `gsutil` to copy the auth file from cloud storage though, so we'd have to provide it manually every time we built the image (at least until the feature starts working). would you rather just do it that way?